dclab/rtdc_dataset/plugins/plugin_feature.py

```diff
@@ -0,0 +1,65 @@
+"""
+.. versionadded:: 0.34.0
+"""
+
+import pathlib
+import importlib
+import sys
+
+from ..ancillaries import AncillaryFeature
+
+
+_registered_plugin_instances = []
```

Comment:
See the `remove_all_plugin_features` function on line 49.

Reply:
I think we don't necessarily need that. All plugin features are instances of `PluginFeature` and are listed in `AncillaryFeature.features`.